agent/consul/state/intention_test.go

```diff
@@ -1,6 +1,8 @@
 package state
 
 import (
+	"github.com/hashicorp/consul/acl"
```

Comment:
I think these should go in the second block of imports.

Reply:
Fixed, this issue with my IDE is getting old